api/payloads/manifest.go

```diff
@@ -89,11 +89,13 @@ func (p ManifestApplicationProcess) ToProcessCreateMessage(appGUID, spaceGUID st
 
 	diskQuotaMB = uint64(1024)
 	if p.DiskQuota != nil {
+		// error ignored intentionally, since the manifest yaml is validated in handlers
```

Comment:
I'm generally anti-comment, but I actually think this one and the other that was removed below are useful and should be kept.